### DISCUSSION TITLE
Perturbation simulation

### DISCUSSION THREAD

**Discussion by gilgolan73** [aertslab/scenicplus#521] [General]
Hello,
I was going through the tutorial for TF perturbation (https://scenicplus.readthedocs.io/en/latest/Perturbation_simulation.html).
I wanted to ask- does the random forest regression model takes into account TFs that are negative regulators of the genes? or only positive regulators? 
Because I see that the gene_to_TF object (given to the train_gene_expression_models function) includes both negative and positive regulators. 

Thank you,
Gil

**Comment by SeppeDeWinter** [comment]
Hi @gilgolan73 

Yes, it does take into account both positive and negative interactions.

Best,

Seppe

**Comment by gilgolan73** [comment]
Hi @SeppeDeWinter ,
Does it distinguish between them ? because the gene_to_TF object includes both of them without any information about whether they positively or negatively regulate the gene.

Thanks,
Gil

**Reply by SeppeDeWinter** [reply]
Hi @gilgolan73 

Sorry I misunderstood your question.
No it treats both kind of TFs in the same way (i.e. this extra information is not explicitly provided as an extra feature). 

Best,

S

**Comment by gilgolan73** [comment]
Hi,
OK. So do you believe it will be better to give the model only positive regulators ? (to get more accurate results for the perturbation simulation).

Thanks,
Gil

**Comment by SeppeDeWinter** [comment]
No, I think giving both is better.
The model will learn (again) that some have a negative effect. This might be important for predicting gene expression.

S

**Comment by gilgolan73** [comment]
OK, thank you for the answer
